entity-server-api/src/main/java/org/terracotta/entity/EntityServerService.java

```diff
@@ -64,12 +64,51 @@
   PassiveServerEntity<M, R> createPassiveEntity(ServiceRegistry registry, byte[] configuration);
   
   /**
+   * Reconfigure an existing entity during server execution.  Reconfigure call in the
+   * platform's entity API allows an entity to reconfigure in place, its instance in the 
+   * system.  Prior to the call of this method, the entities execution queue is flushed and 
+   * any subsequent messages are blocked until this call completes.  This call occurs only 
+   * under the MANAGEMENT_KEY concurrency key.  Implementations are allowed to either reconfigure
+   * in place and return the same instance of the entity or a new entity may be built if desired.
+   * 
+   * @param <AP> the entity type being reconfigured
+   * @param registry the service registry for the entity
+   * @param oldEntity the instance of the entity implementation being reconfigured
+   * @param configuration the new configuration of the entity
+   * @return An entity instance influenced by the new configuration
+   */
+  default <AP extends CommonServerEntity<M, R>> AP reconfigureEntity(ServiceRegistry registry, AP oldEntity, byte[] configuration) {
```

Comment:
Surprised that this is considered ok while the discussions around passive entity becoming active could not happen with such a hand over.
What is the fundamental difference between the two?